### PR TITLE
ThreatQ - minor fixes

### PR DESCRIPTION
**Pull request by dantavori**
## Status
Ready

## Description
- Fixed URLs of objects' details from ThreatQ UI
- Added missing ThreatQ.File context outputs on threatq-search-by-id (same outputs as in threatq-upload-file)
- Added unit-testing

## Does it break backward compatibility?
   - No

### CODE REVIEW

**Comment by Itay4** [comment]
please add test

**Comment by kirbles19** [comment]
@dantavori Reviewed the .yml, but there is no changelog....

**Comment by ronykoz** [comment]
@dantavori please add CHANGELOG

**Comment by dantavori** [comment]
@kirbles19 Updated the changelog, TIA

File: Integrations/ThreatQ_v2/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+  - Fixed results numbering in ***threatq-get-all-adversaries***, ***threatq-get-all-indicators*** and ***threatq-get-all-events*** commands.
+  - Added unit-testing to the integration.
```
Comment (yaakovi) [inline Integrations/ThreatQ_v2/CHANGELOG.md]: this shouldn't be in the change log

Comment (dantavori) [inline Integrations/ThreatQ_v2/CHANGELOG.md]:
```suggestion
```

Comment (dantavori) [reply Integrations/ThreatQ_v2/CHANGELOG.md]: fixed (can be seen under threatq-integration branch)

**Comment by ronykoz** [comment]
@kirbles19 just go over the CHANGELOG here please

**Comment by kirbles19** [comment]
@dantavori @ronykoz Done.

**Pull request merged**